Advanced C# compiler options - PathMap: separator should be a comma instead of a semicolon

- Content: [Advanced C# compiler options](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/advanced#pathmap)
- Content Source: [docs/csharp/language-reference/compiler-options/advanced.md](https://github.com/dotnet/docs/blob/main/docs/csharp/language-reference/compiler-options/advanced.md)
- Product: **dotnet-csharp**
- Technology: **csharp-language-reference**

```
To specify multiple mapped source paths, separate each with a semicolon.

XML
<PathMap>path1=sourcePath1;path2=sourcePath2</PathMap>
```

Using a semicolon will cause the following error.

_CSC : error CS8101: The pathmap option was incorrectly formatted._

It should be a comma.

```
To specify multiple mapped source paths, separate each with a comma.

XML
<PathMap>path1=sourcePath1,path2=sourcePath2</PathMap>
```

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 0db1080c-bbc9-5c15-dea7-cf3c5638ea00
* Version Independent ID: 6a993c1e-13a2-c354-5d3d-4e9d401781b9
* Content: [C# Compiler Options - advanced scenarios](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/advanced#pathmap)
* Content Source: [docs/csharp/language-reference/compiler-options/advanced.md](https://github.com/dotnet/docs/blob/main/docs/csharp/language-reference/compiler-options/advanced.md)
* Product: **dotnet-csharp**
* Technology: **csharp-language-reference**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

--- a/docs/csharp/language-reference/compiler-options/advanced.md
+++ b/docs/csharp/language-reference/compiler-options/advanced.md
@@ -75,10 +75,10 @@ When you specify [**DebugType**](code-generation.md#debugtype), the compiler wil
 
 ## PathMap
 
-The **PathMap** compiler option specifies how to map physical paths to source path names output by the compiler. This option maps each physical path on the machine where the compiler runs to a corresponding path that should be written in the output files. In the following example, `path1` is the full path to the source files in the current environment, and `sourcePath1` is the source path substituted for `path1` in any output files. To specify multiple mapped source paths, separate each with a semicolon.
+The **PathMap** compiler option specifies how to map physical paths to source path names output by the compiler. This option maps each physical path on the machine where the compiler runs to a corresponding path that should be written in the output files. In the following example, `path1` is the full path to the source files in the current environment, and `sourcePath1` is the source path substituted for `path1` in any output files. To specify multiple mapped source paths, separate each with a comma.
 
 ```xml
-<PathMap>path1=sourcePath1;path2=sourcePath2</PathMap>
+<PathMap>path1=sourcePath1,path2=sourcePath2</PathMap>
 ```
 
 The compiler writes the source path into its output for the following reasons:

